docs/creating-fakes.md

```diff
@@ -52,12 +52,8 @@ var foo = A.Fake<FooClass>(x => x.Implements(typeof(IFoo)));
 var foo = A.Fake<FooClass>(x => x.Implements<IFoo>());
 
 // Assigning custom attributes to the faked class.

```

Comment:
I feel slightly bad, but while we're here, `IFoo` is not a class. "faked t ype"?